met-api/src/met_api/resources/widget_documents.py

```diff
@@ -40,7 +38,7 @@ def get(widget_id):
         """Fetch a list of widgets by engagement_id."""
         try:
             documents = WidgetDocumentService().get_document_by_widget_id(widget_id)
-            return json.dumps(documents), HTTPStatus.OK
+            return ActionResult.success(result=documents)
```

Comment:
any reason for adding this ?
I was planning to eventually get rid of actionresult..

Reply:
Because in the front-end we expect: ApiResponse<T = unknown> {
    status: boolean;
    message: string;
    id?: string;
    result?: T;
}

Reply:
We need a separate task to change this design back and front if needed

Reply:
I think we should change it? is it gonna be a big change?
move to the better pattern in new code than go back?
<img width="910" alt="image" src="https://user-images.githubusercontent.com/39633456/203845749-0002af52-31e7-4b3f-8c8a-9d6c0034812b.png">

Reply:
but this is  anew API call in front end? Can we just change for that?

Reply:
if you have it tested well , lets get it merged and revisit it i think

Reply:
Yeah let's discuss this change with Tiago. In the front-end it's not all new

This is new
![image](https://user-images.githubusercontent.com/91914654/203850186-ea4dcf3c-18a8-4497-a968-59d95b5e01d5.png)

but we use api manager which isn't new and expects ApiRespinse
![image](https://user-images.githubusercontent.com/91914654/203850263-cbc40b91-ce18-4bf1-a75f-65ae848da29c.png)

Reply:
![image](https://user-images.githubusercontent.com/91914654/203850322-4eeb2100-e085-49c7-97fc-e22b4b41448e.png)